ansible-ipi-install/roles/installer/templates/metal3-config.j2

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: metal3-config
+  namespace: openshift-machine-api
+data:
+  cache_url: ''
+  deploy_kernel_url: http://{{ prov_ip }}:6180/images/ironic-python-agent.kernel
+  deploy_ramdisk_url: http://{{ prov_ip }}:6180/images/ironic-python-agent.initramfs
+  dhcp_range: 172.22.0.10,172.22.0.100
+  http_port: "6180"
+  ironic_endpoint: http://{{ prov_ip }}:6385/v1/
+  ironic_inspector_endpoint: http://{{ prov_ip }}:5050/v1/
+  provisioning_interface: {{ prov_nic }}
+  provisioning_ip: {{ prov_ip }}/24
```

Comment:
I know the mask is 24 bit for now, but should we make this a variable since they are planning on making this flexible?

Reply:
I think we can revisit this once it is an actual issue since provisioning_ip right now is not modifiable (yet).